riyaz/cli/__init__.py

```diff
@@ -118,7 +117,46 @@ def new_site(path):
     fmt.success(f"New Riyaz site created at {path}")
 
 
-@contextlib.contextmanager
+@main.command("import-course", short_help="load course into a riyaz site")
+@click.option("-s", "--site", default=Path("."), show_default=True,
+              type=click.Path(path_type=Path),
+              help="path to site directory (created with `riyaz new-site`)")
```

Comment:
Do you really want to provide `--site` option? With that you are signing up for making sure all the relative `database_path` continue to work even when you are in a different directory. 

I would suggest that make it compusory to run it from the site directory and not get into those issues at all.

Reply:
I feel it would be too restrictive without it.

After a Riyaz installation, the --site directory is set once and then used while developing other courses. So it's more likely users will remember its path or keep it at a location that can be easily accessed through absolute paths.

Or if there is just one Riyaz site on the machine, we can put it into `~/.riyaz` and then register an alias like:
```shell
alias update-course='riyaz import-course --site ~/.riyaz'
```

> With that you are signing up for making sure all the relative database_path continue to work even when you are in a different directory.

There is an extra step in `config.load_config` where we resolve the paths loaded from YAML: [snippet](https://github.com/pipalacademy/riyaz/commit/11eb177e42fa19838d84a357ef64d0ee2180aabe#diff-913ad1a30eb86e01f56801eacff692d399d91c83350db2f80140a0171b8886cbR17-R25)

What do you think?

Reply:
> I feel it would be too restrictive without it.

I'm not against having that feature, but warning you that it comes with a cost and be ready for it. At least write a test to make sure that works even if you call it from a different directory.

> Or if there is just one Riyaz site on the machine, we can put it into `~/.riyaz` and then register an alias like:
> 
> ```shell
> alias update-course='riyaz import-course --site ~/.riyaz'
> ```

Nope. That is a bad advise. Asking someone to create an alias means that developer experience of the command is poor. 

I would recommend supporting reading that value from an environment variable. I find the term `site` a bit confusing. 

Wouldn't it be better to call it `root` or `-d/ --root-directory` indicating the root directory of the riyaz instance and environment variable `RIYAZ_ROOT`?

> There is an extra step in `config.load_config` where we resolve the paths loaded from YAML: [snippet](https://github.com/pipalacademy/riyaz/commit/11eb177e42fa19838d84a357ef64d0ee2180aabe#diff-913ad1a30eb86e01f56801eacff692d399d91c83350db2f80140a0171b8886cbR17-R25)
> 
> What do you think?

Sounds good.

Reply:
I wrote the tests but there are also lots of unrelated changes in tests.
I'll create a separate PR for fixing the tests.